notebooks/book2/03/linreg_post_pred_plot.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<p>Add meaningful title to notebook, see other closed PR to see how they added title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>